packages/hint-create-element-svg/README.md

```diff
@@ -0,0 +1,80 @@
+# No `createElement` with SVG (`create-element-svg`)
+
+This hint informs users that they need to use [`createElementNS`][createElementNS] to create SVG elements instead of [`createElement`][createElement]
```

Comment:
```suggestion
This hint informs users that they need to use [`createElementNS`][createElementNS] to create SVG elements instead of [`createElement`][createElement].
```